packages/headless/src/app/commerce-engine/commerce-engine.ts

```diff
@@ -59,18 +60,17 @@ export type CommerceEngineState =
  * @internal WORK IN PROGRESS. DO NOT USE IN ACTUAL IMPLEMENTATIONS.
  */
 export interface CommerceEngine<State extends object = {}>
-  extends CoreEngine<State & CommerceEngineState, CommerceThunkExtraArguments> {
+  extends CoreEngineNext<
+    State & CommerceEngineState,
+    CommerceThunkExtraArguments
+  > {
   /**
    * Executes the first search.
-   *
-   * @param analyticsEvent - The analytics event to log in association with the first search. If unspecified, `logInterfaceLoad` will be used.
    */
   executeFirstSearch(): void;
 
   /**
-   * Executes the first search, and logs the analytics event that triggered a redirection from a standalone search box.
-   *
-   * @param analytics - The standalone search box analytics data.
+   * Executes the first search after a redirection from a standalone search box.
```

Comment:
Note to self: given that we're not logging client side events with the commerce sub-package, this actually does the exact same thing as executeFirstSearch... could probably be removed, although I see that @fpbrault added it, so it's probably there for a reason.